instrumentation/apache-httpclient/apache-httpclient-4.0/javaagent/src/test/groovy/ApacheHttpClientResponseHandlerTest.groovy

```diff
@@ -1,55 +0,0 @@
-/*
```

Comment:
I couldn't find a difference between this test and the `ResponseHandler` tests in the other file (now the `doRequestAsync` tests) so deleted, but check my work that it was indeed redundant.